app/services/AuditService.scala

```diff
@@ -112,11 +114,18 @@ class AuditService @Inject()(auditConnector: AuditConnector){
 
     }
 
+    val response = if (trustTaxable.isDefined) {
+      Json.obj(
+        "tvn" -> variationResponse.tvn,
+        "trustTaxable"-> trustTaxable.get
+      )
+    } else {Json.toJson(variationResponse)}
```

Comment:
Minor comment but possibly better practice to pattern match on trustTaxable or fold over it rather than call `.isDefined` and `.get`

Reply:
Updated